### ISSUE TITLE
ci: RUN_UNIT_TESTS_SEQUENTIAL is broken on Windows

### ISSUE THREAD

**Issue by hebasto** [bitcoin/bitcoin#18768]
If the `RUN_UNIT_TESTS_SEQUENTIAL` variable is set to `true` the following message is printed in the Travis log:

```
bash: /home/travis/build/hebasto/bitcoin/build/bitcoin-*/src/test/test_bitcoin: No such file or directory
```

and no tests actually are executed.

Related: #18562

**Comment by MarcoFalke** [comment]
Example, please. Works for me:

* https://travis-ci.org/github/bitcoin/bitcoin/jobs/672571896#L4000
* https://travis-ci.org/github/bitcoin/bitcoin/jobs/679397430#L3994

**Comment by hebasto** [comment]
@MarcoFalke 
> Example, please. Works for me:
> 
>     * https://travis-ci.org/github/bitcoin/bitcoin/jobs/672571896#L4000
> 
>     * https://travis-ci.org/github/bitcoin/bitcoin/jobs/679397430#L3994

https://travis-ci.org/github/hebasto/bitcoin/jobs/679560355#3202

with the following patch:
```diff
diff --git a/ci/test/00_setup_env_win64.sh b/ci/test/00_setup_env_win64.sh
index c3f094bda..8f0c62a1a 100644
--- a/ci/test/00_setup_env_win64.sh
+++ b/ci/test/00_setup_env_win64.sh
@@ -9,8 +9,6 @@ export LC_ALL=C.UTF-8
 export CONTAINER_NAME=ci_win64
 export HOST=x86_64-w64-mingw32
 export PACKAGES="python3 nsis g++-mingw-w64-x86-64 wine-binfmt wine64"
-export RUN_UNIT_TESTS=false
-export RUN_UNIT_TESTS_SEQUENTIAL=true
 export RUN_FUNCTIONAL_TESTS=false
 export GOAL="deploy"
 export BITCOIN_CONFIG="--enable-reduce-exports --disable-gui-tests"
```

**Comment by MarcoFalke** [comment]
Windows executables end in `.exe`

**Comment by MarcoFalke** [comment]
Fixed in #18735
